saleor/graphql/core/fields.py

```diff
@@ -36,231 +26,51 @@ def patch_pagination_args(field: DjangoConnectionField):
     )
 
 
-class BaseConnectionField(graphene.ConnectionField):
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
+class ConnectionField(graphene.Field):
+    def __init__(self, type_, *args, **kwargs):
+        kwargs.setdefault("before", graphene.String())
```

Comment:
If we are defining these fields here, can we also set the descriptions here? Then we could drop `patch_pagination_args`.